src/Validation/Validation.php

```diff
@@ -442,10 +442,6 @@ public static function containsNonAlphaNumeric($check, int $count = 1): bool
      */
     public static function custom($check, ?string $regex = null): bool
     {
-        if (!is_string($check)) {
```

Comment:
This check is intended to keep non-scalar data out as arrays could come in via request data. Could we use `is_scalar()` instead of removing this check?

Reply:
Yes, it's work great. I have patched my forked pull request version.